src/Clay/Render.hs

```diff
@@ -314,12 +314,11 @@ predicate ft = mconcat $
     Class        a   -> [ "." , fromText a                                             ]
     Attr         a   -> [ "[" , fromText a,                     "]"                    ]
     AttrVal      a v -> [ "[" , fromText a,  "='", fromText v, "']"                    ]
-    AttrBegins   a v -> [ "[" , fromText a, "^='", fromText v, "']"                    ]
+    AttrBegins   a v -> [ " [" , fromText a, "^='", fromText v, "']"                    ]
```

Comment:
I'm a bit surprised about this change, in the context of all the other lines that simply have `"["` without the space. Is there a CSS spec somewhere that says that this space must be present?

Reply:
I am not a CSS expert, I just reported the behaviour that doesn't work, so I thought that you are agreeing with some kind of fix :slightly_smiling_face: I can't think of a nicer solution, if you have ideas, I can use them :+1: I agree that this looks like a hack, that is why I am asking if you can see a better way to resolve the problem.

Reply:
I'm probably less of a CSS expert than you even, so if something like this is needed, then we'll do it. My thinking in this case was that maybe the other `"["` need to be changed to `" ["` as well. So to find that out, I was wondering whether maybe you know that or you can point me to a resource so I can look that up :)